travis/test.sh

```diff
@@ -27,3 +23,9 @@ dart dev/tools/mega_gallery.dart
 (cd examples/layers; flutter test)
 (cd examples/material_gallery; flutter test)
 (cd examples/stocks; flutter test)
+
+# end of section that needs to be in sync with the infra bots
```

Comment:
The infra bots shell out to this file now.  Maybe remove these comments about keeping things in sync?